src/components/StorageOverview/Alerts/strings.js

```diff
@@ -0,0 +1,3 @@
+export const ALL = 'All';
```

Comment:
this should probably go under `components/Dashboard`

Reply:
right now `strings.js` file for each component is inside that component so followed the same pattern...if u want i will move it !

Reply:
these constants will be reused by other dashboards, so it would be good if you could move it